src/compute-client/src/controller/sequential_hydration.rs

```diff
@@ -345,7 +345,7 @@ async fn forward_messages<C, T>(
                         break; // client disconnected
                     }
                     Err(error) => {
-                        tracing::error!("error on receive");
+                        tracing::warn!("error on receive");
```

Comment:
@teskje is probably the right person to review this?

Reply:
Hm, these error logs look wrong. We can receive errors here when the client disconnects, which is expected and not something we'd want to be notified about. I suspect I put in the logs while doing some debugging and then forgot to take them out again afterwards 🙈 

@def- Instead of downgrading this log, could you remove it? And also the other two `error!` invocations in this file.